src/commands/step_processor.go

```diff
@@ -85,6 +86,10 @@ func (s *stepSession) processSingleSession() int64 {
 	result, err := s.Client().Installer.GetNextSteps(s.Context(), &params)
 	if err != nil {
 		s.Logger().Warnf("Could not query next steps: %s", err.Error())
+		if reflect.TypeOf(err) == reflect.TypeOf(installer.NewGetNextStepsNotFound()) {
```

Comment:
I think checking the type of the error and relying on the type of the error in order to decide what to do is not a good idea.
It is not part of the protocol that 404 for next-steps is sleep forever.  If someone will see it he will not know why it was done.

Reply:
that's what i was asked too

Reply:
i don't think u need to remind to say the truth. It is not interesting to anyone. 
Or we are sending from bm-inventory sleep forever or just verifying the error here.
It ticket i was asked to verify the error, it is easier but we can open a dispute one more time.

Reply:
OK. Sent this also in the ticket.  If this opinion will persist then we will go on with it.

Reply:
@ronniel1 FYI

Reply:
I think this is fine and expected.  If the cluster no longer exists, there is nothing for it to do.